vnext/Playground/Playground.sln

```diff
@@ -51,50 +52,50 @@ Global
 		{A990658C-CE31-4BCC-976F-0FC6B1AF693D}.Debug|ARM.Build.0 = Debug|ARM
 		{A990658C-CE31-4BCC-976F-0FC6B1AF693D}.Debug|x64.ActiveCfg = Debug|x64
 		{A990658C-CE31-4BCC-976F-0FC6B1AF693D}.Debug|x64.Build.0 = Debug|x64
-		{A990658C-CE31-4BCC-976F-0FC6B1AF693D}.Debug|x86.ActiveCfg = Debug|x86
-		{A990658C-CE31-4BCC-976F-0FC6B1AF693D}.Debug|x86.Build.0 = Debug|x86
+		{A990658C-CE31-4BCC-976F-0FC6B1AF693D}.Debug|x86.ActiveCfg = Debug|Win32
```

Comment:
you probably didn't want to change all these x86's to win32?

Reply:
Weird and unexpected.  Reverted manually.

---
In reply to: [309979579](https://github.com/microsoft/react-native-windows/pull/2862#discussion_r309979579) [](ancestors = 309979579)